src/main/resources/gbd_disability_weights.json

```diff
@@ -538,5 +538,11 @@
     "system": "SNOMED-CT",
     "code": "197927001",
     "disability_weight": 0.051
-    }
+    },
+
+  "Hypothyroidism" : {
```

Comment:
Seeing this here makes me think these disability weights (and other things like costs) should not be split up in this separate file, but defined in the module itself. Right now the module is just the states, but we could easily extend it so that a module could list out the codes it's going to use, and then define costs or disability weights as appropriate. That way the module would be self-contained.